web/js/function-highlight-addon.js

```diff
@@ -0,0 +1,176 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
If you like, you can add a README to that directory describing your changes, and you can also add a copyright mention for "The CodeWorld Authors".  Neither is necessary.